### PR TITLE
Add recipe for adios_db package

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
| Language        | Name of review team           |
| --------------- | ----------------------------- |
| python          | `@conda-forge/help-python`    |
| other           | `@conda-forge/staged-recipes` |

Once the PR is ready for review, please mention one of the teams above in a
new comment. i.e. `@conda-forge/help-some-language, ready for review!`
Then, a bot will label the PR as 'review-requested'.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ChrisBarker-NOAA** [comment]
@conda-forge/help-python, ready for review!

**Comment by JamesMakela-NOAA** [comment]
I am willing to be a maintainer

File: recipes/adios_db/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - adios_db_oil_query = adios_db.scripts.oil_query:oil_query_cmd
+    - adios_db_backup = adios_db.scripts.db_backup:backup_db_cmd
+    - adios_db_restore = adios_db.scripts.db_restore:restore_db_cmd
+    - adios_db_validate = adios_db.scripts.validate:main
+    - adios_db_update_test_data = adios_db.scripts.update_test_data:main
+    - adios_db_process_json = adios_db.scripts.process_json:run_through
+
+requirements:
+  host:
+    - python
```
Comment (BastianZim) [inline recipes/adios_db/meta.yaml:28]:
```suggestion
    - python >=3.8
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
One small thing and this is good to go.

**Comment by ChrisBarker-NOAA** [comment]
thanks! -- I'm always a bit confused about the host section ...

**Comment by ChrisBarker-NOAA** [comment]
Thanks!

**Pull request merged**